tests/kernel/device/src/abstract_driver.c

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2018 Intel Corporation
```

Comment:
change year to 2020

Reply:
Has changed the year to 2020 for three new added files, thanks!